test/collection.js

```diff
@@ -1355,6 +1355,14 @@
     ok(collection.comparator === comparator);
   });
 
+  test("Pass `null` for `models` for empty Col with `options`", 2, function() {
```

Comment:
0 is probably overdoing it. Keep it with false, undefined, and null, and I'll merge. Thanks!